auslib/web/public/base.py

```diff
@@ -0,0 +1,97 @@
+import cgi
+import connexion
+import logging
+
+from flask import make_response, send_from_directory, Response
+
+from raven.contrib.flask import Sentry
+
+from auslib.AUS import AUS
+from auslib.dockerflow import create_dockerflow_endpoints
+from auslib.web.api_validator import BalrogParameterValidator
+
+log = logging.getLogger(__name__)
+
+validator_map = {
+    'parameter': BalrogParameterValidator
+}
+
+connexion_app = connexion.App(
+    __name__, specification_dir='.', validator_map=validator_map)
```

Comment:
Ah! I saw that you had asked about it above, but I had understood another thing.